docs/oras-auth-provider.md

```diff
@@ -0,0 +1,144 @@
+# ORAS Authentication Provider
+
+The ORAS referrer store handles all registry and referrer resolution operations. ORAS provides authentication credentials to access referrer artifacts. Ratify contains many Authentication Providers to support different authentication scenarios. The user specifies which authentication provider to use in the configuration.
```

Comment:
nit: Will ORAS be used to access non-registry referrer operations? Sorry, it might be my understanding. The first line felt like ORAS referrer store for handling registry and referrer operations. May be,  do you want to clarify that it handles referrer  operations using registry as the referrer store?

Reply:
Do ORAS provides auth credentials or user provide through config ?